### DISCUSSION TITLE
Notifications

### DISCUSSION THREAD

**Discussion by anonimitoraf** [200ok-ch/organice#793] [Q&A] [answered]
Hello!

For PC, I rely on https://github.com/spegoraro/org-alert to tell me what I need to do from day-to-day. Does organice offer something similar? Or at least something in the roadmap? (I can implement it myself if there's a clear path forward)

**Comment by munen** [comment, marked as answer]
Hi @anonimitoraf 

Thank you for posting this question. I totally understand where you are coming from - personally, I also want to have notifications on my phone. Since iOS does not support Web notifications (however, it will in the upcoming iOS 16 :partying_face:), I have not implemented it yet. This is how I currently get notifications on my phone. However, it requires a setup off organice: https://200ok.ch/posts/2022-02-13_integrating_org_mode_agenda_into_other_calendar_apps.html

Since iOS 16 will support web notifications (and Android has for a long time, same as Desktop browsers), this is absolutely a feature that should be high up on the road map!

In the back of my mind, I have been thinking about how to do it for a while and came up with this: Notifications can similarly be managed as properties or tags. Sometimes you want one notification, sometimes many. Additionally, it would be useful to have a default configuration of notifications (in the settings screen) that can easily be added in one click. I'm misusing the properties screen here for a demo. Of course, it should be a separate one. What do you think?

![Screen Shot 2022-06-13 at 14 56 05](https://user-images.githubusercontent.com/505721/173358629-d5e5a53a-dd99-449b-8f92-137207a49233.png)

Since you offered to work on it, I'd be happy to take you up on that. If you agree that the UX can be similar to tags or properties, then the implementation is pretty straightforward. Would you still be willing to work on that? I'd be happy to support with testing and a code review in any case!

**Comment by afonsofrancof** [comment]
@munen iOS 16.4 beta 1 introduced support for web notifications. Just a heads up :)
